as.py

```diff
@@ -91,7 +91,7 @@ def parse_args(argv):
     arg_parser.add_argument("--elasticsearch_excluded_observatories", type=str,
                             dest='elasticsearch_excluded_observatories',
                             help="Elasticsearch telemetry observatories to exclude (comma delimited)",
-                            default=defaults.elasticsearch_excluded_observatories)
+                            default=','.join(defaults.elasticsearch_excluded_observatories))
```

Comment:
Double checking - I think `defaults.elasticsearch_excluded_observatories` is a comma delimited string here? Not sure if the `.join()` is doing something unexpected.

Reply:
so this is a tricky one. The environment variable for this is a comma delimited string, but it is converted into list within the SchedulerInput() class because that is how it is used later. The defaults are made by instantiating the SchedulerInput() class, so its a list of strings there, so this converts that back to a comma delimited list, to then be passed back into the SchedulerInput() class initialization. This probably seems pretty convoluted, but this seemed like the best way to support environment variable input, or command line input - it's actually always using command line input, it's just the defaults are derived from the environment variables now.

Reply:
Oh I see... Yeah I was only looking at the diff so missed the part where it's converted into a list. Sounds good!